.github/workflows/close_pr.yml

```diff
@@ -0,0 +1,55 @@
+name: Label closed PR as merged and leave a comment
+on:
+  push
+
```

Comment:
Do we need permissions declared here?

Reply:
Thanks for the callout - this worked fine in my fork, but it looks like that's based on the default permissions of GITHUB_TOKEN, which can be configured differently per-repo (also, if any permissions are declared, the rest are unset). I agree it's better to be explicit either way, so I've added the necessary ones